Library/Formula/dependency-check.rb

```diff
@@ -25,6 +25,10 @@ def install
     (etc/"dependencycheck").mkpath
     corejar = libexec/"repo/org/owasp/dependency-check-core/#{version}/"\
       "dependency-check-core-#{version}.jar"
+    if File.exist? etc/"dependencycheck/dependencycheck.properties"
```

Comment:
The overwrite happens on the following line. The default configuration is a properties file in the JAR - the installer extracts it as a matter of convenience so that the user has a place to modify the configuration that's picked up by default. We unzip that file out on the following line - in the bump from 1.2.11 to 1.3.0, new properties were added, so I want to make sure we get the new options.